projects/igniteui-angular/src/lib/calendar/calendar.ts

```diff
@@ -326,6 +334,26 @@ export class Calendar {
         return result;
     }
 
+    public getDatePerMonthView(date: Date, interval: string, monthViewIdx: number) {
```

Comment:
This function name does not represent what it does, because it does not return a date PER a given month, it always returns the date for the FIRST month view in the calendar. IMO it should be renamed to smth like "getFirstMonthViewDate" or "getDateFirstMonthView"